crates/bevy_app/src/app.rs

```diff
@@ -837,6 +837,10 @@ impl App {
         self
     }
 
+    /// Adds an `App` as a child of the current one.
+    ///
+    /// The `World` of the sub app is the same of the main app. The provided `f` function runs each
```

Comment:
> The `World` of the sub app is the same of the main app.
 
 Is this true? I thought we had to copy data for entities between the main world and render world?

Reply:
Yeah you're right, I got confused. The subapp has its own world, however, the main world is passed as an argument to the World parameter of `f`, so the two can interact.